### ISSUE TITLE
IntelliJ plugin: "Variable expected" error involving multidimensional index operator in assigment

### ISSUE THREAD

**Issue by rsmckinney** [manifold-systems/manifold#412]
"Variable expected" error involving multidimensional array assignent.
```java
foo[3][4] = "hi"; // intellij error on foo[3][4]: Variable expected
```

**Comment by rsmckinney** [comment]
Fix available with IJ plugin release 2022.x.34

**Comment by rsmckinney** [comment]
Note, this fixes issue mentioned in #390
